### PR TITLE
Add UserProfile View

### PR DESCRIPTION
**Pull request by ace0625**
- Add UserProfile view
- Add experience listview in userprofile page
- Change navigator to router: Since router is a lot  easier to implement and manage, decided to change

### CODE REVIEW

File: src/components/UserList/Row.js [hunk 1]
```diff
@@ -5,24 +5,30 @@ import {
...
     return (
-      <View style={styles.row}>
-          <Image style={styles.photo}
-                 source={{ uri: 'http://lorempixel.com/64/64/cats/' }} />
-          <View style={styles.imageSeperator}></View>
-          <View style={styles.userInformation}>
+      <TouchableHighlight onPress={Actions.userProfile}>
+        <View style={styles.row}>
+            <Image style={styles.photo}
+                   source={{ uri: 'http://lorempixel.com/64/64/cats/' }}/>
```
Comment (ethkim) [inline src/components/UserList/Row.js:20]: Leave a TODO to replace this with data from backend.

File: src/components/UserList/Row.js [hunk 2]
```diff
@@ -5,24 +5,30 @@ import {
...
+            <View style={styles.userInformation}>
 
-            {/* Below Texts are mock data.
-             They'll be replaced with real data from backend */}
-            <Text style={styles.name}>Hyunchan Kim</Text>
-            <Text style={styles.job}>software engineer</Text>
-            <Text style={styles.education}>education</Text>
-          </View>
-      </View>
+              {/* Below Texts are mock data.
```
Comment (ethkim) [inline src/components/UserList/Row.js]: Add "TODO"

File: src/components/UserList/UserList.js [hunk 3]
```diff
@@ -22,8 +23,11 @@ class UserList extends Component {
     this.state = {
 
       // Initialize with mock data for testing listview.
-      dataSource: dataSource.cloneWithRows([1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5,1, 2, 3, 4, 5]),
+      dataSource: dataSource.cloneWithRows([1, 2, 3, 4, 5, 1, 2, 3, 4, 5,
```
Comment (ethkim) [inline src/components/UserList/UserList.js]: Let's clean this up a little:

```
  dataSource: dataSource.cloneWithRows([
    1, 2, 3, 4, 5, 1, 2, 3, 4, 5,
    1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 
    1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 
    1, 2, 3, 4, 5,]),
```

File: src/components/userProfile/UserProfile.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    StyleSheet,
+    View,
+    Text,
+    ScrollView,
+    Image,
+    TouchableHighlight,
+    Dimensions,
+    ListView,
+} from 'react-native';
+import { Actions, } from 'react-native-router-flux';
```
Comment (ethkim) [inline src/components/userProfile/UserProfile.js]: Is it customary to add comma , after Actions? Just curious

**Comment by ethkim** [comment]
Mostly style comments. LGTM
Thanks Hyunchan!

File: src/components/userProfile/UserProfile.js (hunk 4 above)
Comment (ace0625) [reply src/components/userProfile/UserProfile.js]: Mostly, it's required to place a comma on import or some other parts even if there's nothing after a comma. However, it seems okay without a comma on that part.

**Comment by ifndefdeadmau5** [comment]
LGTM

**Pull request merged**